src/libtcod/renderer_xterm.c

```diff
@@ -36,20 +36,32 @@
 #include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <pthread.h>
```

Comment:
If you wanted this to support Windows then you might need to avoid `pthread.h`. SDL has threading if you want to use those.

Reply:
I've switched it to `SDL_CreateThread()`.